docs/README.md

```diff
@@ -0,0 +1,20 @@
+# Goal
+
+Create a fully open-source learning resource for young and old.
+
+Giving access to everyone to learn at their own pace and in their own way.
+<!-- Too many courses are behind a pay-wall, to only provide limited value. -->
+
+## Project goals
+
+Simplicity is key, the simpler our application, the better.
+Instead of focussing on unwanted features and the possibilities of what users would want. It is more important to focus on great UX (user experience) and use that as a growth mechanism.
```

Comment:
```suggestion
Instead of focusing on unwanted features and the possibilities of what users would want, it is more important to focus on great UX (user experience) and use that as a growth mechanism.
```